app/models/subscription.rb

```diff
@@ -58,7 +58,9 @@ def subscribe!(brand, subscription_plan, email, stripe_token)
       if customer
         stripe_subscription_id = customer.subscriptions.first.id
         create_payment_handler!(brand, customer)
-        create_subscription!(brand, subscription_plan, stripe_subscription_id, trial_end)
+        subscription = create_subscription!(brand, subscription_plan, stripe_subscription_id, trial_end)
+        UserMailer.welcome(brand)
```

Comment:
If you want to have an easy API for triggering these mailers with the new `TransactionalEmail` service is to have singleton methods that pass in the required merge vars. So, it could be

```ruby
TransactionalEmail.welcome(brand)
```

where

```ruby
module TransactionalMailers
  def welcome(brand)
    self.new(
      self::Types::WELCOME,
      brand,
      [{ name: 'FOO_BAR', content: 'I am the content for foobar' }],
    )
  end
end

class TransactionalEmail
  extend TransactionalMailers
end
```

or something like that